### PR TITLE
Spelling & grammar fixes within mixins

### PR DESCRIPTION
**Pull request by melchiar**
Various spelling and grammatical corrections mostly to jsdoc descriptions, although a few are variable/function names (let me know if any of them would be considered breaking changes).

### CODE REVIEW

**Comment by melchiar** [comment]
Ugh... e87e23b and 7e19de9 somehow managed to stowaway in this PR. Sorry, still getting the hang of github. Is there an easy way to revert those while keeping 565e6c3?

File: src/mixins/canvas_events.mixin.js [hunk 1]
```diff
@@ -750,7 +750,7 @@
      * @param {String} config.evtIn name of the event to fire for in
      * @private
      */
-    fireSynteticInOutEvents: function(target, e, config) {
+    fireSyntheticInOutEvents: function(target, e, config) {
```
Comment (asturur) [inline src/mixins/canvas_events.mixin.js]: ehhh english.

**Comment by asturur** [comment]
we have still the same problem of before. Mixed commits ( shadowUniform is here )

**Comment by asturur** [comment]
you should have your master aligned with fabricJS master ( or some commit behind )
To quickfix it you can enter this branch and use:

`git checkout master src/shapes/object.class.js`

and 

`git checkout master src/shadow.class.js`

then commit and push.

**Comment by asturur** [comment]
hey thank you!

**Pull request merged**